src/tribler-core/tribler_core/modules/metadata_store/community/remote_query_community.py

```diff
@@ -0,0 +1,87 @@
+import json
+from binascii import unhexlify
+from random import sample
+
+from ipv8.community import Community
+from ipv8.lazy_community import lazy_wrapper
+from ipv8.messaging.lazy_payload import VariablePayload
+from ipv8.peer import Peer
+from ipv8.requestcache import RequestCache
+
+from tribler_core.modules.metadata_store.orm_bindings.channel_metadata import entries_to_chunk
+
+minimal_blob_size = 200
```

Comment:
Some preemption for testing and the future: please put these global variables into some sort of "settings" object (like https://github.com/Tribler/py-ipv8/blob/master/ipv8/attestation/trustchain/settings.py).

Reply:
Good point. However, I would prefer to keep it simple for now, so how about just moving these into `__init__` instance variables?

Reply:
Sure, that's a start.